installer/win/WixSharpInstaller/CloseDlg.cs

```diff
@@ -0,0 +1,57 @@
+using WixSharp;
+using System.Drawing;
+
+namespace Symphony
+{
+    public partial class CloseDlg : WixSharp.UI.Forms.ManagedForm, IManagedDialog
```

Comment:
can we refactor this to be called `CloseDialog` instead of the short form?

Reply:
Keeping it consistent with the already added dialogs for now. We can change it, but that would be a separate improvement that we should apply to all dialogs in that case.